backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/TelemetryProperties.kt

```diff
@@ -0,0 +1,29 @@
+package io.tolgee.configuration.tolgee
+
+import io.tolgee.configuration.annotations.DocProperty
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "tolgee.telemetry")
+@DocProperty(
+  description = """By default, self-hosted instances send anonymous data about usage to help us improve Tolgee.
+
+:::info
+This was added in Tolgee Platform v3.23.0
+:::
+
+Once a day we collect following data
+- number of projects
+- number of languages
+- number of translations
+- number of users
+
+We don't collect any other data. Please leave telemetry enabled to help us improve Tolgee.""",
+  displayName = "Telemetry"
+)
+class TelemetryProperties {
+  @DocProperty(description = "Whether telemetry is enabled")
+  var enabled: Boolean = true
+
+  @DocProperty(hidden = true)
+  var server: String = "https://app.tolgee.io"
```

Comment:
This seems to be used only once and never modified. Should it be a constant instead, or is the property here in case E2E tests need them?

Reply:
it's modified in tests, search for `tolgee.telemetry.server`

Reply:
I must be blind, because I double checked and I don't see it being used besides in the TelemetryService... The tests do modify whether it is enabled and the reporting time in ms, but I couldn't find an instance of `server` being modified/altered 🤔